lib/containers/k8s.pm

```diff
@@ -103,6 +103,11 @@ sub install_k3s {
         delete $k3s_args{$key};
     }
 
+    if (is_tumbleweed && !is_microos) {
```

Comment:
Can be moved inside the block for `K3S_INSTALL_UPSTREAM`

Reply:
True. Fixed.